.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Run Unit Tests
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - '**'
+
+concurrency:
+  group: ${{ github.ref }}-unit-tests
+  cancel-in-progress: true
+
+jobs:
+  unit-tests:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '18.x'
```

Comment:
Can we have a build matrix (i.e more versions of node to run on). At least 3 back + latest. 18 seems old?